packages/waci-pex/src/agent/qr-utils/QrPropValue.tsx

```diff
@@ -0,0 +1,28 @@
+import {AuthenticationRequestURI} from "@sphereon/did-auth-siop/dist/main/types/SIOP.types";
+import {SIOP, RP} from "@sphereon/did-auth-siop";
+import {SsiQrCodeProps} from "../../types/ssi-qr-code-provider-types";
+
+export abstract class QrPropValue {
+
+  public static qrValue(ssiQrCodeProps: SsiQrCodeProps): string {
+    let qrValue: string = '';
+
+    QrPropValue
+      .url(ssiQrCodeProps.authenticationRequestOpts)
+      .then(value => qrValue = value.encodedUri);
+
+    return qrValue;
+  }
+
+  private static async url(authenticationRequestOpts: SIOP.AuthenticationRequestOpts): Promise<AuthenticationRequestURI> {
+
```

Comment:
empty line can be removed

Reply:
The vertical spacing is in my opinion a very useful tool to signify what is a new aspect. But for now removed it.